plugins/filter/jc.py

```diff
@@ -94,7 +94,12 @@ def jc(data, parser, quiet=True, raw=False):
         dictionary or list of dictionaries
 
     Example:
-
+        # This step is optional in case you have the package already installed on your machine
+        - name: add the jc prereqs to the running machine
```

Comment:
Note that this is a task, while the next list element is a play. This doesn't really fit well together.